clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus.go

```diff
@@ -59,6 +59,16 @@ const (
 	// countQuery %v should be replaced with (1) filters and (2) query window size.
 	countQuery = "sum(increase(apiserver_request_duration_seconds_count{%v}[%v])) by (resource, subresource, scope, verb)"
 
+	countSlowQuery = "sum(rate(apiserver_request_duration_seconds_bucket{%v}[%v])) by (resource, subresource, scope, verb)"
+
+	// Bucket definitions: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go#L84
+	// exclude all buckets of 1s and shorter
+	filterGetAndMutating = filters + `, verb!="LIST", le!~"0.\\d+|1"`
+	// include all buckets above 5s
```

Comment:
This is a bit error prone, if buckets will change in k/k.
I would prefer removing all buckets <=5s (even though the regexp will be slightly more difficult)
I think it will be something
"0.\\d+|[1234](.\\d+)*|5"